.github/ISSUE_TEMPLATE/usage_feedback.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Usage feedback
+about: Let us know if gnss_lib_py has been helpful to you!
+title: ''
+labels: usage feedback
+assignees: ''
+
+---
+
+**Feedback**
+How has `gnss_lib_py` been helpful to you?
```

Comment:
I think we should also ask if people would recommend changing anything or if they're using gnss_lib_py for a specific application or course

Reply:
additional questions added